### ISSUE TITLE
a2 question e miss a small quenstion

### ISSUE THREAD

**Issue by jcjview** [LooperXX/CS224n-2019#1]
Very nice HW!

in a2 question e ,it need a small written quenstion.
> After you’ve done this, describe with one sentence why this loss function is much more efficient to compute than the naive-softmax loss.
